### DISCUSSION TITLE
Pass parameter to cmake

### DISCUSSION THREAD

**Discussion by Kurochka** [Futureblur/magnet#25] [Q&A]
Hi. For example, I would like to build something like CTranslate2 (https://opennmt.net/CTranslate2/installation.html), but it depends on oneDnn(https://github.com/oneapi-src/oneDNN). So, I need `cmake -DWITH_DNNL=ON ..`. How can I do it with magnet?

**Comment by rdsov** [comment]
You should be able to pass down any arguments to magnet commands.

So something like `magnet build -DWITH_DNLL=ON` should work.

If you have a dependency, you need to get it first: `magnet pull https://github.com/oneapi-src/oneDNN.git`
